addons/mail/static/src/core/web/recipients_input_tags_list_popover.js

```diff
@@ -28,8 +28,11 @@ export class RecipientsInputTagsListPopover extends Component {
         });
     }
 
+    /**
+     * @deprecated
+     */
     get id() {
-        return this.props.tagToUpdate.id;
+        return this.props.tagToUpdate.resId;
```

Comment:
Since this is being deprecated do we really need to change it then?

Reply:
I suppose it doesn't, but I also think the original purpose for this getter was to have the resId, since it is a more useful information, isn't it?